lib/inspec/runner.rb

```diff
@@ -89,7 +89,12 @@ def tests
     end
 
     def configure_transport
-      @backend = Inspec::Backend.create(@conf)
+      backend = Inspec::Backend.create(@conf)
+      set_backend(backend)
+    end
+
+    def set_backend(new_backend)
```

Comment:
Could  add a comment saying it is a powerful method and will have consequences - this may break RunData, for example, or require resetting the RSpec world.

Reply:
I'll set aside this concern for now. We haven't thought of a single concrete example of or a single broken test.